ydb/library/yql/yt/native/plugin.cpp

```diff
@@ -313,8 +314,45 @@ class TYqlPlugin
             ActiveQueriesProgress_[queryId].Compiled = true;
         }
 
+        NYql::NEmbedded::EExecuteMode executeModeEnum;
+
+        // Transforming NYT::NYqlClient::EExecuteMode (yt/yt/ytlib/yql_client/public.h)
+        // to NYql::EExecuteMode
+        switch (executeMode) {
+        case 0:
+            executeModeEnum = NYql::NEmbedded::EExecuteMode::Validate;
+            break;
+        case 1:
+            executeModeEnum = NYql::NEmbedded::EExecuteMode::Optimize;
+            break;
+        case 2:
+            executeModeEnum = NYql::NEmbedded::EExecuteMode::Run;
+            break;
+        case 3:
+            executeModeEnum = NYql::NEmbedded::EExecuteMode::Lineage;
+            break;
+        default:
+            return TQueryResult{
+                .YsonError = MessageToYtErrorYson(Format("Unknown execution mode: %v", executeMode)),
+            };
+        }
+
         NYql::TProgram::TStatus status = NYql::TProgram::TStatus::Error;
-        status = program->RunWithConfig(impersonationUser, pipelineConfigurator);
+        switch (executeModeEnum) {
```

Comment:
Maybe we can use just one switch and write something like:
```
case 0: // NYql::NEmbedded::EExecuteMode::Run:
    status = program->RunWithConfig(impersonationUser, pipelineConfigurator);
    break;
```

It won't change the logic, but it will simplify the code a little bit.

Reply:
Original idea was to transform `NYT::NYqlClient::EExecuteMode` to `NYql::NEmbedded::EExecuteMode`, and then cover all cases in "switch"

Seems like we don't need to use `NYql::NEmbedded::EExecuteMode` at all, so removed this logic and use one switch now